WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/UseCases/RefundCreationUseCase.swift

```diff
@@ -58,6 +62,30 @@ struct RefundCreationUseCase {
                             total: calculateTotal(of: refundable),
                             totalTax: "")
         }
+
+        if let shippingLine = shippingLine {
+            refundItems.append(createShippingItem(from: shippingLine))
+        }
+
+        return refundItems
+    }
+
+    /// Returns an `OrderItemRefund` based on the provided `ShippingLine`
+    ///
+    private func createShippingItem(from shippingLine: ShippingLine) -> OrderItemRefund {
+        OrderItemRefund(itemID: shippingLine.shippingID,
```

Comment:
just curious, could this item ID collide with an order item's ID from `createRefundItems` above? does the API have a way to differentiate a refund item between `ShippingLine` and an order item?

Reply:
That is an interesting option, I've created a [task](https://github.com/woocommerce/woocommerce-ios/projects/17#card-48092261) to ask core about this possibility.